src/components/SendFormModal/SendFormModal.tsx

```diff
@@ -91,15 +102,16 @@ export const SendFormModal: FunctionComponent<SendFormModalProps> = (props) => {
     if (nextStage !== undefined) {
       if (nextStage !== Stages.SBA_PCR_REVIEW && assignee !== undefined) {
         // All stages other than SBA PCR require the assignee to be defined/selected
-        await props.submit(nextStage, assignee, noteText);
+        await props.submit(nextStage, assignee, noteText, false);
         closeForm();
       } else if (
         nextStage === Stages.SBA_PCR_REVIEW &&
         pcrEmail &&
-        checkSBAPCRValid(pcrEmail) === undefined
+        checkSBAPCRValid(pcrEmail) === undefined &&
+        (greaterThan35MB ? ackManualSend : true)
```

Comment:
It isn't quite the same.  In the case that it wasn't greater than 35MB, neither side would be true, so it wouldn't proceed, but it should as they don't need to check that they've manually sent.  It could be written as greaterThan35MB === ackManualSend.  So that if both are false or both are true then we proceed.   However, that might be more confusing, so I think the way it is written is probably best.  If we are greater than 35MB, then we need to also have the checkbox checked, otherwise, if we aren't greater than 35MB, then proceed.

Reply:
🤦‍♂️ -- yes, that would work much better - I'll update it

Reply:
Updated